nginx.conf

```diff
@@ -38,15 +38,26 @@ http {
   			proxy_pass http://$VulnerableApplication:80;
         }
         
-        location /VulnerabilityDefinitions {
+        location ~ ^/(?:VulnerabilityDefinitions|scanner\/sast)$ {
         	default_type 'application/json';
         	content_by_lua_block {
-        		local vulnerableAppResponse = ngx.location.capture("/VulnerableApp/VulnerabilityDefinitions")
-                local vulnerableAppJspResponse = ngx.location.capture("/VulnerableApp-jsp/VulnerabilityDefinitions")
-        		local vulnerableAppPhpResponse = ngx.location.capture("/VulnerableApp-php/VulnerabilityDefinitions")
+        		local vulnerableAppResponse = ngx.location.capture("/VulnerableApp" .. ngx.var.uri)
+                local vulnerableAppJspResponse = ngx.location.capture("/VulnerableApp-jsp" .. ngx.var.uri)
+        		local vulnerableAppPhpResponse = ngx.location.capture("/VulnerableApp-php" .. ngx.var.uri)
 				ngx.say(require("vulnerableapp_utility").merge_vulnerability_information(vulnerableAppResponse, vulnerableAppJspResponse, vulnerableAppPhpResponse))
     		}
         }
+
+		location /scanner/dast {
```

Comment:
please correct the indentation.

Reply:
In my vs code it has correct indentation, will push again to after changes.

Reply:
For some reason, the indentation changes ... after pushing up

Reply:
okay, remotely editing the file fixed this issue